### ISSUE TITLE
haproxy-certbot keeps restating 

### ISSUE THREAD

**Issue by raheem52** [thingsboard/thingsboard#1550]
I am trying to launch things board as mca docker containers
From docker folder, I am launching them by  "sh docker-start-services.sh "

All containers are launching properly but the container with haproxy named as "haproxy-certbot" keeps restarting.

When I checked container logs using  "logs -f haproxy-certbot" I get this error in log :

Adding qdisc plug dev lo id 40: parent 1:4
Error: Unable to add qdisc: Object not found
  refcnt 0no options

**Comment by vipcgh** [comment]
me too

**Comment by raheem52** [comment]
Was trying this on mac and was restarting but when I did so in Ubuntu it worked.
Not sure what exactly the issue is

**Comment by Schelling1989** [comment]
Facing the same issue...

**Comment by danielgreenx** [comment]
me too!
I am trying to deploy tb using docker-compose scripts on minikube (in win 10).
Have you solved the problem?

**Comment by toti1212** [comment]
+1

**Comment by raheem52** [comment]
I didn't worked for me on Mac .. Then I used Ubuntu on VirtualBox ..where it worked

**Comment by kryptonn** [comment]
Same for me on a mac... anyone solve this issue ?

Macos Mojave 10.14.6 (18G95)

**Comment by gsugambit** [comment]
im having this problem in ubuntu. what's weird is i had to build the image on mac because it would not build on ubuntu. it kept freezing on the make file :(

**Comment by zshwei** [comment]
same problem even in 2022 ...

**Comment by 2817748290** [comment]
+1

**Comment by mgulden** [comment]
I have same problem on
RHEL 8
Docker CE 24.0.7
Thingsboard 3.4.4 (Docker compose setup uses haproxy 1.3, and i also tried 1.7.0 both same result)

**Comment by smatvienko-tb** [comment]
The latest version was successfully started on MacBook Air with Apple silicone M1

TLDR: removed `tc` `qdisc` commands in the scripts

Please use that versions:
```
docker pull thingsboard/haproxy-certbot:2.2.33-alpine
```
or the same with another tag 
```
docker pull thingsboard/haproxy-certbot:1.9.0
```

![image](https://github.com/thingsboard/thingsboard/assets/79898499/79bc37d6-f9c3-4dd1-8801-36908e6cba12)
